dali/pipeline/workspace/host_workspace.cc

```diff
@@ -18,24 +18,23 @@
 
 namespace dali {
 
-void HostWorkspace::GetSample(SampleWorkspace *ws,
-    int data_idx, int thread_idx) {
+void HostWorkspace::GetSample(SampleWorkspace* ws, int data_idx, int thread_idx) {
   DALI_ENFORCE(ws != nullptr, "Input workspace is nullptr.");
   ws->Clear();
   ws->set_data_idx(data_idx);
   ws->set_thread_idx(thread_idx);
-  for (const auto &input_meta : input_index_map_) {
+  for (const auto& input_meta : input_index_map_) {
```

Comment:
FFFUUUUUUUUUUU

Reply:
https://clang.llvm.org/docs/ClangFormat.html

Reply:
FFFUUUU is reformatting otherwise unchanged code.